client/state/signup/progress/reducer.js

```diff
@@ -1,6 +1,6 @@
 import debugFactory from 'debug';
 import { has, keyBy, get, omit } from 'lodash';
-import stepsConfig from 'calypso/signup/config/steps-pure';
+import stepsConfig from 'calypso/signup/config/steps';
```

Comment:
If we use `steps-pure`, it cannot get correct apiRequestfunction so that the status of step would mark as `completed` and skip API request.

Reply:
I think this change is what's caused the unit test failures. `steps-pure` doesn't pull in any networking code so it can run in a pure node environment.